scripts/utils/execute.js

```diff
@@ -0,0 +1,82 @@
+const path = require('path');
+const execa = require('execa');
+
+/**
+ * Groups all string arguments into a single one. E.g.:
+ * ```js
+ * ['-m', '"Upgrade:', 'to', 'latest', 'version"'] --> ['-m', '"Upgrade: to latest version"']`
+ * ```
+ *
+ * Original code writeng by @molant for webhint/hint.
+ * Distributed under an Apache-2.0 License
```

Comment:
Probably for another PR, but I realize that we have no License specified for this project.

Reply:
I've opened #31 for this.